packages/kolibri-tools/test/fixtures/TestUserPermissions.js

```diff
@@ -63,6 +65,8 @@ function fetchUserPermissions(userId) {
  * @returns Promise<void>
  */
 export function showUserPermissionsPage(store, userId) {
+  const { getfacilities } = useFacilities();
```

Comment:
This should be `getFacilities` - maybe that's why you had to add the import above?